tests/test_store_durability.py

```
@@ -0,0 +1,453 @@
+
+@pytest.mark.parametrize('always_race', [False, True])
+def test_inspection_retries_changed_source_instead_of_reporting_corruption(
+        tmp_path, monkeypatch, always_race):
+    import skodun.store as mod
+    db = tmp_path / 'healthy.db'
+    _write_review(db, 'retained')
+    real_snapshot = mod._snapshot_database
+    copies = []
+    def snapshot(path):
+        result = real_snapshot(path)
+        temporary, image, error = result
```

Ruff (0.16.3)

[warning] Unpacked variable `temporary` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)